code/modules/mob/living/taste.dm

```diff
@@ -21,6 +21,8 @@
 		var/text_output = from.generate_taste_message(taste_sensitivity)
 		// We dont want to spam the same message over and over again at the
 		// person. Give it a bit of a buffer.
+		if(hallucination > 50 && prob(25))
+			text_output = pick("spiders","[pick(wire_colors)]","dreams","nightmares","the future","the past","victory","defeat","pain","bliss","revenge","poison","time","space","death","life")
```

Comment:
the american way
also add everything listed here in this thread: https://tgstation13.org/phpBB/viewtopic.php?f=55&t=9256